tests/e2e/performance/test_fio_benchmark.py

```diff
@@ -13,37 +15,62 @@
 from ocs_ci.ocs.ripsaw import RipSaw
 from ocs_ci.ocs import constants, node
 from ocs_ci.utility.performance_dashboard import push_perf_dashboard
-from ocs_ci.framework.testlib import E2ETest, performance
+from ocs_ci.framework import config
+from ocs_ci.framework.testlib import E2ETest, performance, skipif_ocs_version
 from ocs_ci.ocs.perfresult import PerfResult
 from ocs_ci.ocs.elasticsearch import ElasticSearch
-from ocs_ci.ocs.cluster import CephCluster
+from ocs_ci.ocs.cluster import CephCluster, calculate_compression_ratio
 from ocs_ci.ocs.version import get_environment_info
 
 log = logging.getLogger(__name__)
+dev_mode = config.RUN["cli_params"].get("dev_mode")
+ERRMSG = "Error in command"
 
 
-@pytest.fixture(scope="function")
-def es(request):
+def run_command(cmd):
```

Comment:
why do we need a new function for run_command?
can't we use utils.run_cmd?

Reply:
This function is consuming less memory then utils.run_cmd.
there is also a PR to make this function generic for performance purports #3572.

Reply:
Discussed offline and the merge into utils run_cmd will be done separately